it/repositories/JourneyDataRepositoryISpec.scala

```diff
@@ -16,42 +16,80 @@
 
 package repositories
 
+import assets.TestConstants.{testInternalId, testJourneyId}
 import org.scalatest.concurrent.{AbstractPatienceConfiguration, Eventually}
 import org.scalatest.time.{Seconds, Span}
 import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.libs.json.{JsString, Json}
 import play.api.test.Helpers._
 import play.api.{Application, Environment, Mode}
 import uk.gov.hmrc.incorporatedentityidentification.models.IncorporatedEntityIdentificationModel
-import utils.ComponentSpecHelper
-import assets.TestConstants.testJourneyId
 import uk.gov.hmrc.incorporatedentityidentification.repositories.JourneyDataRepository
-import utils.WiremockHelper.scaled
+import utils.ComponentSpecHelper
 
 import scala.concurrent.ExecutionContext.Implicits.global
 
-class JourneyDataRepositoryISpec extends ComponentSpecHelper with AbstractPatienceConfiguration with Eventually{
-
-    override lazy val app: Application = new GuiceApplicationBuilder()
-      .in(Environment.simple(mode = Mode.Dev))
-      .configure(config)
-      .configure("application.router" -> "testOnlyDoNotUseInAppConf.Routes")
-      .configure("mongodb.timeToLiveSeconds" -> "10")
-      .build
-
-    val repo: JourneyDataRepository = app.injector.instanceOf[JourneyDataRepository]
-
-    "documents" should {
-      "expire" in {
-        val journeyId = testJourneyId
-        val journeyConfig = "continueURL"
-        await(repo.createJourney(journeyId, None))
-        implicit val patienceConfig: PatienceConfig =
-          PatienceConfig(timeout = scaled(Span(90, Seconds)), interval = scaled(Span(10, Seconds)))
-        eventually {
-          await(repo.findById(journeyId)) mustBe None
-        }
+class JourneyDataRepositoryISpec extends ComponentSpecHelper with AbstractPatienceConfiguration with Eventually {
+
+  override lazy val app: Application = new GuiceApplicationBuilder()
+    .in(Environment.simple(mode = Mode.Dev))
+    .configure(config)
+    .configure("application.router" -> "testOnlyDoNotUseInAppConf.Routes")
+    .configure("mongodb.timeToLiveSeconds" -> "10")
+    .build
+
+  val repo: JourneyDataRepository = app.injector.instanceOf[JourneyDataRepository]
+
+  override def beforeEach: Unit = {
+    super.beforeEach()
+    await(repo.drop)
+  }
+
+  val authInternalIdKey: String = "authInternalId"
+  val creationTimestampKey: String = "creationTimestamp"
+
+  "documents" should {
+    "expire" in {
```

Comment:
Would be good if this test checks the TTL index is there rather than checking that mongo deletes the document